packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -471,6 +523,29 @@ async function updateAutoLink(
       });
     }
 
+    if (addCsModuleSupport) {
+      // Add managed projects
+      projectsForSolution.push({
+        projectFile: path.join(
+          rnwRoot,
+          'Microsoft.ReactNative.Managed/Microsoft.ReactNative.Managed.csproj',
+        ),
+        projectName: 'Microsoft.ReactNative.Managed',
+        projectLang: 'cs',
+        projectGuid: '{F2824844-CE15-4242-9420-308923CD76C3}',
+      });
+      projectsForSolution.push({
+        projectFile: path.join(
+          rnwRoot,
+          'Microsoft.ReactNative.Managed.CodeGen//Microsoft.ReactNative.Managed.CodeGen.csproj',
+        ),
+        projectName: 'Microsoft.ReactNative.Managed.CodeGen',
+        projectLang: 'cs',
+        projectGuid: '{ADED4FBE-887D-4271-AF24-F0823BCE7961}',
+        projectTypeGuid: '{9A19103F-16F7-4668-BE54-9A1E7A4F7556}',
```

Comment:
consider making these consts
also why is the project above not have a project type guid but this one does?

Reply:
All C++ projects use one guid for project type. All "normal" c# projects use one guid for project type.

dotnet core (for the codegen project) uses a different one.